### PR TITLE
refactor(docs): convert to <AngleBrackets>

### PR DESCRIPTION
**Pull request by mehrdadrafiee**
This PR converts {{curly-braces}} to <AngleBrackets> syntax in the docs. ~~For some reason the new syntax was not working on some of Ember CLI AddonDocs' components .~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "babel-eslint": "^10.0.1",
     "broccoli-asset-rev": "^2.7.0",
     "chai": "^4.1.2",
+    "cz-conventional-changelog": "^3.1.0",
```
Comment (alexlafroscia) [inline package.json:29]: Can you remove this part? It doesn’t seem relevant to the rest of the PR

**Comment by alexlafroscia** [comment]
Hey @mehrdadrafiee -- I'm actually working on a major re-factoring to modernize this add-on and use Glimmer under-the-food. I'm going to close this PR and cherry-pick your commits into the PR that I'm working on, so that I can get those updates in there.

**Comment by mehrdadrafiee** [comment]
@alexlafroscia sounds good to me 👍 
sorry for the late reply. got slammed with work :(

**Comment by alexlafroscia** [comment]
No worries! I totally understand

**Pull request closed without merging**